packages/ses/README.md

```diff
@@ -140,8 +141,8 @@ console.log(Object.isFrozen(capability.inc));
 
 Note that although the **surface** of the capability is frozen, the capability
 still closes over the mutable counter.
-Hardening an object graph makes the surface immutable, but does not make
-methods pure.
+Hardening an object graph makes the surface immutable, but does not guarantee
+that methods are free of side effects.
```

Comment:
Worth working in here that the purpose of `freeze` and `harden` is not primarily about immutability, but primarily about tamper proofing API surfaces.